README.md

```diff
@@ -288,29 +288,17 @@ OpenMock leverages [https://golang.org/pkg/text/template/](https://golang.org/pk
   {{.HTTPBody | xmlPath "node1/node2/node3"}}
   ```
 
-# Admin Interface
-Openmock also opens an admin interface on port 9998, which allows you to see and 
-manipulate the loaded templates.
+## Command Line Interface
+Openmock has a command-line interface to help with certain tasks interacting with openmock instances. This is 
+invoked with the `omctl` command.  This uses the [cobra](https://github.com/spf13/cobra) library to provide a discoverable CLI; run `omctl` for a list of commands / flags. 
 
-## List Templates
-`curl -X GET http://localhost:9998/api/v1/templates`
-Prints YAML representation of all the templates the instance has loaded
-
-## Update Templates
-```
-curl -Xcurl -X POST \
-  http://localhost:9998/api/v1/templates \
-  -d '<template yaml>'
-```
-Add template(s) (per the posted YAML) to the instance
-
-### Post helper
-There is an additional executable provided by openmock to help post templates
-at a remote instance.  Use the command: 
+### CLI: Directory
+#### Update Directory
+Updates the openmock directory on a remote openmock instance by loading a directory from the local filesystem, and posting it to the remote instance.
 ```
-OPENMOCK_TEMPLATES_DIR=./demo_templates OPENMOCK_ENDPOINT=http://localhost:9998 om_post_templates
+# Adds templates from the ./demo_templates directory to the instance running on localhost.
+omctl directory update --directory ./demo_templates --url http://localhost:9998
```

Comment:
is this the definition update? what about `omctl definition|behavior|template update --directory ...  `

Reply:
Yes, would love to work on the terminology here.  
RE: UPDATE: yes, this is the utility to open openmock files locally and POST them at the admin interface of a remote openmock instance.